bindata/network/openshift-sdn/rules.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+  prometheus: k8s
+  role: alert-rules
+  name: networking-rules
+  namespace: openshift-sdn
+spec:
+  groups:
+  - name: general.rules
+    rules:
+    - alert: NodeWithoutOVSPod
+      annotations:
+        message: |
+          All nodes should be running an ovs pod. One or more are not
+      expr: |
+        :kube_pod_info_node_count: != sum(kube_pod_container_info{namespace="openshift-sdn", pod=~"ovs.*"})
```

Comment:
Is there a way to alert on the specific node(s) missing the pod?

Reply:
Yes this is possible, you can use the `kube_pod_info` metric (with the appropriate selector) and join them onto the `kube_node_info` metric. That way we'll be able to figure out which nodes have such a pod and which doesn't.